pycbc/events/stat.py

```diff
@@ -1542,8 +1546,23 @@ def rank_stat_coinc(self, s, slide, step, to_shift,
         """
 
         sngl_rates = {sngl[0]: sngl[1]['snglstat'] for sngl in s}
-        ln_noise_rate = coinc_rate.combination_noise_lograte(
-                                  sngl_rates, kwargs['time_addition'])
+        if 'dets' in kwargs:
+            ln_noise_rate = coinc_rate.combination_noise_lograte(
+                                    sngl_rates, kwargs['time_addition'],
+                                    kwargs['dets'])
+
+            # Find total volume of phase-time-amplitude space occupied by
+            # noise coincs
+            # Extent of time-difference space occupied
```

Comment:
This comment seems to apply to both branches of the if statement, so maybe move it outside?

Reply:
Suggested change in another thread fixes this